examples/api/lib/material/radio_list_tile/custom_labeled_radio.1.dart

```diff
@@ -23,43 +23,21 @@ class LabeledRadioApp extends StatelessWidget {
 }
 
 class LabeledRadio extends StatelessWidget {
-  const LabeledRadio({
-    super.key,
-    required this.label,
-    required this.padding,
-    required this.groupValue,
-    required this.value,
-    required this.onChanged,
-  });
+  const LabeledRadio({super.key, required this.label, required this.padding, required this.value});
 
   final String label;
   final EdgeInsets padding;
-  final bool groupValue;
   final bool value;
-  final ValueChanged<bool> onChanged;
 
   @override
   Widget build(BuildContext context) {
     return InkWell(
       onTap: () {
-        if (value != groupValue) {
-          onChanged(value);
-        }
+        RadioGroup.maybeOf<bool>(context)?.onChanged(value);
```

Comment:
This looks good now, thanks for refactoring! So LabeledRadio will work like any other Radio button now.